spec/requests/providers/application_merits_task/opponent_types_controller_spec.rb

```diff
@@ -61,9 +61,9 @@ module ApplicationMeritsTask
           let(:radio_button) { "false" }
 
           # TODO: This should redirect to the organisation opponent page when AP-4301  is complete.
```

Comment:
Can we now remove this TODO?

Reply:
@naseberry you are looking at an early /outdated version of the code, this has changed in later commits.